R/RPPA_19Q1.R

```diff
@@ -1,13 +1,13 @@
-#' RPPA
+#' RPPA_19Q1
```

Comment:
See general comment: only one .R file per data object with a documentation header for all releases.